src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -185,7 +217,11 @@ const ProfileSelector = ({ input }) => {
   );
 };
 
-const OScapNoneOption = ({ setProfileName }) => {
+const OScapNoneOption = ({
+  setProfileName,
+}: {
+  setProfileName: (name: string) => void;
+}) => {
```

Comment:
We usually define a type that holds the info and reference that instead of typing directly.
Could be cool to keep the pattern that is used in the rest of code base.

```
type OScapNoneOptionPropType = {
  setProfileName: (name: string) => void;
}
```

I agree that's practically equivalent as the other option you wrote, but let's have all the code having the same vibe.